proposal.rst

```diff
@@ -166,27 +185,35 @@ Project Details
 
 Describe any existing work and concepts on which your project is based.
 
-FILLME
+1. https://github.com/danielkorzekwa/tennis-rating-dbn-em-scala
+It is a tennis ratings model based on Dynamic Bayesian Networks and Expectation Maximization, implemented in Scala.
+I'll refer to it to estimate γ in DBN with EM in C++.
+
+2. https://github.com/varepsilon/clickmodels
+It is a set of click models implementations including Dependent Click Model and Intent Aware Models.
+I'll refer to it to add DCM and IAM.
 
 Do you have any preliminary findings or results which suggest that your
 approach is possible and likely to succeed?
 
-FILLME
+As the papers of DBN, DCM and IAM stated. 
 
 What other approaches to have your considered, and why did you reject those in
 favour of your chosen approach?
 
-FILLME
+DBN, DCM and IAM were come up many years ago. Thus, I may also want to follow investigate some recent works to improve omega results while I havent found suitable ones yet.
 
 Please note any uncertainties or aspects which depend on further research or
 investigation.
 
-FILLME
+DBN, DCM and IAM were come up many years ago. Thus, I may also want to follow investigate some recent works to improve omega results while I havent found suitable ones yet.
 
```

Comment:
That's the exact same answer for two questions in a row.  If you don't have an answer, better to just say so that cut and paste the same text.